viewer/packages/utilities/src/customObject/CustomObjectIntersectInput.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { Raycaster, Vector3, Vector2, PerspectiveCamera, Plane } from 'three';
+
+/**
+ * This class is used as input for intersecting custom objects
+ * @beta
+ */
+export class CustomObjectIntersectInput {
+  readonly normalizedCoords: Vector2;
+  readonly camera: PerspectiveCamera;
+  readonly clippingPlanes: Plane[] | null;
```

Comment:
I'd prefer using `undefined` instead of `null` here

Reply:
Fixed